src/TLuaInterpreter.cpp

```diff
@@ -10037,6 +9991,61 @@ int TLuaInterpreter::getMapLabel(lua_State* L)
     return 1;
 }
 
+void TLuaInterpreter::pushMapLabelPropertiesToLua(lua_State* L, TMapLabel label)
```

Comment:
It would be more efficient to pass the label details as a pointer or a reference - this would remove the need for the code to make a copy of it to create the `label` here - as, given the structure of a `TMapLabel` it isn't obvious to me that Qt's CopyOnWrite/Implicit sharing mechanism will work for them...

Reply:
Like that?
```void TLuaInterpreter::pushMapLabelPropertiesToLua(lua_State* L, TMapLabel& label)```

Reply:
Yeah, but make it a `const` reference as well - because you are then accessing the **original** item and you do **not** want to modify it here (by accident)!

Reply:
Done :)